### DISCUSSION TITLE
Should we close the old forum and use this?

### DISCUSSION THREAD

**Discussion by DominusExult** [exult/exult#262] [Polls]
The forum is slow and most features are missing because SourceForge doesn't allow them anymore. I'm toying with moving the websiteand forum but that would add another thing to manage and I'm not 100% looking forward to it.

So I'm looking at the github discussions and not really disliking it. 
Labels, mark helpful answers, file attachments, notifications, better mobile view, polls...etc.
This would also take a burden off my shoulders to maintain and keep the forum up to date.
This forum would then be set to read only. NOT deleted!

What do you think?

**Comment by drcode1** [comment]
I agree.

**Comment by KnightCaptainU7** [comment]
Let's see how this goes. Looks like my old GitHub account was deactivated for inactivity.

**Reply by KnightCaptainU7** [reply]
And I cannot reset it to bring it back by myself, so I emailed in. I have a few items I could add to SIfixes.

**Comment by KnightCaptainU7** [comment]
"GitHub no longer accepts requests to release dormant usernames ."

**Reply by DominusExult** [reply]
but you are not trying to claim a dormant username, you are trying to get your username back! There must be something you can do. What is the problem? The email address no longer available?

**Comment by KnightCaptainU7** [comment]
I think once they mark it as inactive, it is now permanent. This is to avoid the cost to customer service.

"GitHub no longer accepts requests to release dormant usernames .
It appears that you are looking to contact us regarding releasing a dormant or inactive username. We no longer review individual requests to release dormant or inactive usernames."

**Comment by KnightCaptainU7** [comment]
This has been a good move. It makes it easier to share images in posts. And with my own exult-art repo lets me share things quickly.
